packages/ui-portal/src/Portal/index.tsx

```diff
@@ -22,25 +22,29 @@
  * SOFTWARE.
  */
 
-import React, { Component } from 'react'
+import React, { Component, HTMLAttributes } from 'react'
 import PropTypes from 'prop-types'
 import ReactDOM from 'react-dom'
 
 import { element } from '@instructure/ui-prop-types'
 
 import { ReactPortal } from './ReactPortal'
 import { SubtreePortal } from './SubtreePortal'
+import { BidirectionalProps } from '@instructure/ui-i18n'
+
+import { PortalNode } from './PortalType'
 
 const IS_CREATE_PORTAL_SUPPORTED = typeof ReactDOM.createPortal === 'function'
```

Comment:
As I see this is supported since React 16, so we can delete SubTreePortal 
https://newbedev.com/how-to-use-reactdom-createportal-in-react-16
https://instructure.atlassian.net/browse/INSTUI-2287

Reply:
Nice catch! Removed it and simplified the Portal to use the native React portal.